README.md

```diff
@@ -2,13 +2,16 @@
 
 This mini-challenge is intended to test your knowledge of C programming. There are 13 C programming questions that can be found in the challenge.c file. Your solution to this challenge will be verified automatically on a standard Linux machine once you make a pull request.
 
-If you'd like to test your solution locally, make sure you have Make and GCC for your system installed.
+To test your solutions locally, you can use the provided Dockerfile to build a Docker image that will run the tests for you.
+ 
+To build the Docker image, run:
+```
+docker build -t fw-onboarding-level1 .
+```
 
-To test your solution, run:
-```sh
-make clean
-make all
-./build/challenge # Or ./build/challenge.exe on Windows
+To run the tests, run:
+```
+docker run --rm -it fw-onboarding-level1
```

Comment:
Can you write a Notion doc on installing and setting up Docker and link that here? I'd assume most new members won't already have Docker.

Reply:
Notion doc added [here](https://www.notion.so/uworbital/How-To-Docker). It's a fairly quick introduction to understand the commands used in the Dockerfile, plus a general introduction and overview of Docker.

Reply:
I can't seem to edit the [Firmware Onboarding](https://www.notion.so/uworbital/Firmware-Onboarding) page to add a link to the How-To though.

Reply:
you should be able to now

Reply:
Yup, added it to the list of How-Tos as well.